src/app/auth/services/session-storage.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SessionStorageService {
+  constructor(private window: Window) {}
```

Comment:
It's better to inject Document and talk to window on it, because it will abstract actual implementation from runtime.

In case you will move application to server you won't have a headache how to mock window object which is not present during server rendering